src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -72,9 +73,8 @@ const _ChatBox: React.FC<ChatboxProps> = ({
     async ({ text, threadId }) => {
       await sendThreadMessage({ text, threadId });
       unselectOption();
-      closeThread();
     },
-    [unselectOption, closeThread, sendThreadMessage]
```

Comment:
I still think you should leave this, because the requested behaviour was to `unselect thread` after you send a message into the thread

Reply:
@mike-lvov, I've moved `closeThread` to `unselectOption` function which is called here too.

Reply:
Ah, my bad. Looks good then!